### DISCUSSION TITLE
Where to even start?

### DISCUSSION THREAD

**Discussion by jrgiacone** [firefly-iii/firefly-iii#5349] [Firefly III]
I have opened my accounts and credit cards. I have also made a revenue account from my employer (which shows negative). I am coming from mint and ynab, wanting to learn a new process, I am just having trouble figuring out how I am supposed to manage my money with this tool. Does any one have any tips for getting set up in the right way?

**Comment by JC5** [comment]
It looks like its set up already. Why not start adding transactions?
